javascript/javascript2/week3/functionDifferentWays.html

```diff
@@ -0,0 +1,91 @@
+<head>
+    <title>functionDifferentWays</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+</head>
+
+<body>
+    <main>
+
+
+
+
+    </main>
+    <script>
+        /*Create funtions that are used in these different ways:
+
+                        Create an array with 3 items. All items should be functions. Iterate through the array and call all the functions.
+                        Create a function as a const and try creating a function normally. Call both functions. Read up on this if you are interested: https://stackoverflow.com/questions/1013385/what-is-the-difference-between-a-function-expression-vs-declaration-in-javascrip
+                        Create an object that has a key whose value is a function. Try calling this function.*/
+
+
+
+
+        //Function exercise 1
+
+
+        function firstFun() {
+            console.log('I am a first function')
+        }
+
+        function secondFun() {
+            console.log('I am a second function')
+        }
+
+        function thirdFun() {
+            console.log('I am a third function')
+        }
+
+
+        var myArr = [
+            firstFun,
+            secondFun,
+            thirdFun
+        ]
+
+        myArr.forEach(function(func) {
+            func();
+        });
+
+
+        //Function exercise 2
+
+        const myFunction = () => {
+            console.log("My function as const");
+        }
+
+        function myFunction() {
+            console.log("My function, created normally");
+        }
+
+
+
+        //Functions exercise 3
+
+        const obj = {
+            1: function() {
+                first_function(console.log('firstFunction'))
+
```

Comment:
There is no function with a name first_function, this is why it gives an error.
It would be enough just to do like this:
```
1: function() {
                console.log('firstFunction')
}
```
and then call it obj[1]();